store/store.go

```diff
@@ -68,10 +68,11 @@ func NewDatabase(tExpire, tReplicate, tRepublish time.Duration) *Database {
 	db.tExpire = tExpire
 	db.tReplicate = tReplicate
 	db.tRepublish = tRepublish
+	db.setReplicate()
```

Comment:
Was this the bug?

Reply:
Seems like it. No wierd behaviour after I set this value. Still not sure exactly how but it seems it interferes somehow.

Reply:
I suppose replicate would be set to 1 Jan (year 1 ad), if its not called? Does that explain anything?

Reply:
Not completely. That would make the replication run once every second. 
It would explain it all however if the call do  ```db.ch <- remoteItem.Value``` blocks. Which it shouldn't do even if there is no remoteItems in the map, right?

Reply:
It will block if there is no receiver.

Reply:
Hmm, I have ot declared a recieving channel in any of the tests that i dont exlicitly test the channel. This might be the bug then?